src/main/scala/com/linkedin/feathr/offline/mvel/MvelUtils.scala

```diff
@@ -15,9 +16,9 @@ private[offline] object MvelUtils {
   // This approach has pros and cons and will likely be controversial
   // But it should allow for much simpler expressions for extracting features from data sets whose values may often be null
   // (We might not want to check for null explicitly everywhere)
-  def executeExpression(compiledExpression: Any, input: Any, resolverFactory: VariableResolverFactory, featureName: String = ""): Option[AnyRef] = {
+  def executeExpression(compiledExpression: Any, input: Any, resolverFactory: VariableResolverFactory, featureName: String = "", mvelContext: Option[FeathrExpressionExecutionContext]): Option[AnyRef] = {
```

Comment:
Can we make featureName an Option[String]? It looks odd having a default for non-final parameter (especially the various other classes that are calling in with `""` now)

Reply:
SG